src/common/SPELLS/PALADIN.js

```diff
@@ -464,4 +469,137 @@ export default {
     name: 'Blessing of the Ashbringer',
     icon: 'inv_sword_2h_artifactashbringer_d_01',
   },
+
+	// Protection
+  ARDENT_DEFENDER: {
+    id: 31850,
+    name: 'Ardent Defender',
+    icon: 'spell_holy_ardentdefender',
+  },
+  AVENGERS_SHIELD: {
+    id: 31935,
+    name: 'Avenger\'s Shield',
+    icon: 'spell_holy_avengersshield',
+  },
+  CONSECRATION_CAST: {
+    id: 26573,
+    name: 'Consecration',
+    icon: 'spell_holy_innerfire',
+  },
+  GUARDIAN_OF_ANCIENT_KINGS: {
+    id: 86659,
+    name: 'Guardian of Ancient Kings',
+    icon: 'spell_holy_heroism',
+  },
+  HAMMER_OF_THE_RIGHTEOUS: {
+    id: 53595,
+    name: 'Hammer of the Righteous',
+    icon: 'ability_paladin_hammeroftherighteous',
+  },
+  LIGHT_OF_THE_PROTECTOR: {
+    id: 184092,
+    name: 'Light of the Protector',
+    icon: 'ability_paladin_lightoftheprotector',
+  },
+  SHIELD_OF_THE_RIGHTEOUS: {
+    id: 53600,
+    name: 'Shield of the Righteous',
+    icon: 'ability_paladin_shieldofvengeance',
+  },
+  //artifact
+  EYE_OF_TYR: {
+    id: 209202,
+    name: 'Eye of Tyr',
+    icon: 'inv_shield_1h_artifactnorgannon_d_01',
+  },
+	//Talents that become spells
```

Comment:
Do these differ from what's in common/SPELLS/TALENTS? If yes then ok, or if it makes sense to rename it then fine. But otherwise if it's exactly the same, I think it's redundant to have it here again. Also, wouldn't the parser give an error for duplicate keys? https://github.com/WoWAnalyzer/WoWAnalyzer/blob/c83426f31fc7055fced66cc90fe434dac6dbfb81/src/common/safeMerge.js#L8

Reply:
i was under the impression when i first came that even if its a talent, we should add it to the spell list? If we should just simply refer to the talents we will do that. I will ask in discord.

Reply:
You do not add it if it has same spellID has the talent